packages/block-editor/src/components/off-canvas-editor/update-attributes.js

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
Ignore this. It's copy/pasted.

Reply:
Should we add a note to the code about that?